Mathlib/Logic/Equiv/Basic.lean

```diff
@@ -798,7 +805,6 @@ def piCongrLeft : (∀ a, P (e a)) ≃ ∀ b, P b :=
 /-- Note: the "obvious" statement `(piCongrLeft P e) f b = f (e.symm b)` doesn't typecheck: the
 LHS would have type `P b` while the RHS would have type `P (e (e.symm b))`. For that reason,
 we have to explicitly substitute along `e (e.symm b) = b` in the statement of this lemma. -/
-@[simp]
```

Comment:
I think we should keep this lemma, but perhaps use `cast` on the RHS instead?

Reply:
AFAICT, if this lemma is in the default `simp` set, then it fails to simplify `piCongrLeft P e f (e a)`.